### PR TITLE
Fixed multiple trello issues

### PR DESCRIPTION
**Pull request by nullpixel**
yes

### CODE REVIEW

**Comment by nullpixel** [comment]
[Users can approve their own feedback](https://trello.com/c/XrfwBJtJ/37-users-can-approve-their-own-feedback)

File: Commands/Commands/utils.js [hunk 1]
```diff
@@ -311,8 +312,8 @@ commands['deny'] = {
             })
             return
         }
-        if (state[content[0]].deniedUsers.indexOf(message.author.id) > -1) {
-            message.reply('You\'ve already denied this report.').then(delay(config.timeouts.messageDelete)).then((msg) => {
+        if (state[content[0]].deniedUsers.indexOf(message.author.id) > -1 || state[content[0]].approvedUsers.indexOf(message.author.id) > -1) {
```
Comment (Dougley) [inline Commands/Commands/utils.js]: Same as above

File: Commands/Commands/utils.js [hunk 2]
```diff
@@ -243,8 +243,8 @@ commands['approve'] = {
                 })
                 return
             } else {
-                if (state[content[0]].approvedUsers.indexOf(message.author.id) > -1) {
-                    message.reply('You\'ve already approved this report.').then(delay(config.timeouts.messageDelete)).then((msg) => {
+                if (state[content[0]].deniedUsers.indexOf(message.author.id) > -1 || state[content[0]].approvedUsers.indexOf(message.author.id) > -1) {
```
Comment (Dougley) [inline Commands/Commands/utils.js:260]: The point of letting you deny the report even tough you approved it was to let you change your mind, this works (somewhat) the same way on Discord Testers too.

**Comment by nullpixel** [comment]
[Users can approve and deny the same report](https://trello.com/c/DXQ5ZxDN/38-users-can-approve-and-deny-the-same-report)

**Review by Dougley — CHANGES_REQUESTED** [review]
no

**Comment by nullpixel** [comment]
[Users can do as many approvals as they want on one report](https://trello.com/c/gHP5YPQD/39-users-can-do-as-many-approvals-as-they-want-on-one-report)

File: Commands/Commands/utils.js (hunk 2 above)
Comment (nullpixel) [reply Commands/Commands/utils.js:260]: well, the current way won't work for that then. the report will have both an approval, and a deny in the state. discord testers also takes my approach, and you cannot change your mind.

**Comment by nullpixel** [comment]
[The error message for an incorrect report ID is wrong on !deny](https://trello.com/c/651ZTDWn/41-the-error-message-for-an-incorrect-report-id-is-wrong-on-deny)

**Comment by nullpixel** [comment]
[You can mark the same suggestion as duplicates of each other.](https://trello.com/c/hoGEEJ6x/42-you-can-mark-the-same-suggestion-as-duplicates-of-each-other)

**Comment by nullpixel** [comment]
[!duplicate command should error if two URLs are not used.](https://trello.com/c/beqGOTnk/45-duplicate-command-should-error-if-two-urls-are-not-used)

**Review by Dougley — APPROVED** [review]

**Pull request merged**